src/wallet/ismine.cpp

```diff
@@ -3,13 +3,11 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#include <script/ismine.h>
+#include <wallet/ismine.h>
 
 #include <key.h>
-#include <keystore.h>
 #include <script/script.h>
-
```

Comment:
we dont have #include <script/sign.h> in line 10 (as in upstream) So two lines are deleted